eppoclient/configurationrequestor.go

```diff
@@ -59,8 +60,19 @@ func (cr *configurationRequestor) fetchConfig() (configResponse, error) {
 		return configResponse{}, err
 	}
 
+	if cr.skipDeserializeAndUpdateFlagConfigIfUnchanged {
+		if result.ETag == cr.storedUFCResponseETag {
```

Comment:
We should use `If-None-Match` header on the request, so the server can respond with 304 Not Modified and skip sending the body, instead of doing this check on the client side